test/signs/bus_test.exs

```diff
@@ -71,6 +71,7 @@ defmodule Signs.BusTest do
   defmodule FakeConfig do
     def sign_config("auto_sign"), do: :auto
     def sign_config("off_sign"), do: :off
+    def sign_config("Silver_Line.headway"), do: :headway
```

Comment:
This doesn't need this specific ID format anymore, right?

Reply:
Correct. I'll change it to just be `headway`.